source/uwp/Renderer/lib/AdaptiveRenderArgs.h

```diff
@@ -31,3 +30,4 @@ namespace AdaptiveCards { namespace Rendering { namespace Uwp
 
     ActivatableClass(AdaptiveRenderArgs);
 }}}
+#pragma once
```

Comment:
?

Reply:
I have no idea? Thanks for catching it

---
In reply to: [154391331](https://github.com/Microsoft/AdaptiveCards/pull/1013#discussion_r154391331) [](ancestors = 154391331)